milv.config.yaml

```diff
@@ -0,0 +1,13 @@
+#TODO: remove https://dashboard.kyma-project after it's available
+external-links-to-ignore: [ "localhost", "kyma.local", "https://dashboard.kyma-project", "CLUSTER_NAME.kyma.cluster.cx", "$yourClusterDomain", "svc.cluster.local:8081", "https://goreportcard.com", "azure.microsoft.com", "$KYMA_RELEASE_VERSION", "$DOMAIN", "$LATEST", "http://xip.io", "https://storage.{DOMAIN}/minio", "https://$", "http://azure.com", "https://YOUR_OIDC_COMPLIANT_IDENTITY_PROVIDER_INSTANCE/.well-known/openid-configuration" ]
```

Comment:
I guess all these links are appearing in the Kyma docs. We do not need them here.